Automattic/wp-calypso
# Add launchpad site preview dynamic height + lint

#### Proposed Changes

* Changes to make the site preview height dynamic with a min-height (for desktop device only).
* Fixes for stylelint issues

#### Testing Instructions



* Checkout branch and run `yarn start`
* Navigate to the launchpad page: http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug={SITE_SLUG_HERE}
* Zoom out/in on your browser and adjust browser viewport to see site preview expand/shrink.


#### Pre-merge Checklist






Related to #66575
